src/main/java/appeng/block/AEBaseBlock.java

```diff
@@ -584,4 +585,22 @@ public ItemMeshDefinition getItemMeshDefinition()
 		return null;
 	}
 
+	/**
+	 * @return Null if no special block color handler is needed, otherwise the handler to register.
+	 */
+	@SideOnly( Side.CLIENT )
+	public IBlockColor getBlockColor()
+	{
+		return null;
+	}
+
+	/**
+	 * @return Null if no special item color handler is needed, otherwise the handler to register.
+	 */
+	@SideOnly( Side.CLIENT )
+	public IItemColor getItemColor()
```

Comment:
Then there's something wrong with item model. Just look at `BlockColors.register` and you'll understand why.